transforms/code/license_copyright_removal/ray/Makefile

```diff
@@ -0,0 +1,60 @@
+
+# Define the root of the local git clone for the common rules to be able
+# know where they are running from.
+REPOROOT=../../../..
+include $(REPOROOT)/transforms/.make.transforms
+
+TRANSFORM_NAME=license_copyright_removal
+# $(REPOROOT)/.make.versions file contains the versions
+DOCKER_IMAGE_VERSION=${LICENSE_COPYRIGHT_REMOVAL_VERSION}
+
+venv::	.transforms.ray-venv
+
+test::	.transforms.ray-test
+
+clean:: .transforms.clean
+
+image:: .transforms.ray-image
+
+test-src:: .transforms.test-src
+
+setup:: .transforms.setup
+
+test-image:: .transforms.ray-test-image
+
+build:: build-dist image
+
+publish:: publish-dist publish-image
+
+publish-image:: .transforms.publish-image-ray
+
+# distribution versions is the same as image version.
+set-versions:
+	$(MAKE) TOML_VERSION=$(DOCKER_IMAGE_VERSION) .defaults.update-toml
```

Comment:
This should be replaced with a 1-liner.  See noop/ray/Makefile:set-versions,

```
set-versions: 
	$(MAKE) TRANSFORM_PYTHON_VERSION=${NOOP_PYTHON_VERSION} TOML_VERSION=$(NOOP_RAY_VERSION) .transforms.set-versions 
```

Although TOML_VERSION it not needed.